packages/sampler/stories/qa-stories/ExpandableList.js

```diff
@@ -73,24 +80,18 @@ storiesOf('ExpandableList', module)
 		)
 	)
 	.add(
-		'with multiples (to test "lockBottom" prop)',
+		'with default selected',
 		() => (
-			<div>
-				<ExpandableList title="First">
-					{['One', 'Two', 'Three']}
+			<Scroller style={{height: 600}}>
```

Comment:
Should we use ri or base on current window size?

Reply:
Should probably use ri. It _should_ work without a height because both participate in flex layout but it was sizing too large so I gave it a fixed height to avoid the mess.

Reply:
Found the cause. Sampler injects a DOM node in the Panel body that was interfering with the Scroller's height determination. Added a guard around that (which I'm not sure we still use anyway) to prevent rendering it when not valued.

Sadly, this will likely fail @Sangwook-Lee's TAS tests since all the samples will shift up some pixels but it's the right change to make.